static/js/components/JobDetails.jsx

```diff
@@ -1,13 +1,33 @@
 var AddForm = React.createClass({
 
   getInitialState: function() {
-    return {clicked: false, errored: false};
+    return {clicked: false, errored: false, JSON_workload_checked: false};
```

Comment:
The javascript package is `JSON`, so kept in line with that convention, but for variable names we can keep it lower cased.